src/snyk/snykOss/views/suggestion/ossSuggestionWebviewProvider.ts

```diff
@@ -35,6 +37,24 @@ export class OssSuggestionWebviewProvider extends WebviewProvider<OssIssueComman
     );
   }
 
+  async postLearnLessonMessage(vulnerability: OssIssueCommandArg): Promise<void> {
+    if (this.panel) {
+      const learnService = new LearnService(vulnerability, OpenCommandIssueType.OssVulnerability, this.logger);
+      const lesson = await learnService.getLesson();
+      if (lesson) {
+        void this.panel.webview.postMessage({
+          type: 'setLesson',
+          args: { url: `${lesson.url}?loc=ide`, title: 'Learn about this vulnerability' },
```

Comment:
Can we extract the title to /messages/ folder? We try to keep possibly translatable strings there, in case we do translation in future.